portfolio/src/main/webapp/script.js

```diff
@@ -31,10 +31,27 @@ function addRandomGreeting() {
  * Adds random fact about me to the page
  */
 function addRandomFact() {
+  // Keep track of how many times the function has been called, to determine
+  // when to reveal the quiz
+  if (typeof addRandomFact.numTimesCalled === undefined) {
+    addRandomFact.numTimesCalled = 0;
+  }
```

Comment:
I think this could be avoided if we initialize addRandomFact.numTimesCalled

Reply:
I wasn't entirely sure how to pull this off due to the dynamic nature of the page, should I initialize upon loading the page? Or upon loading the `<div>` element where `addRandomFact` is called?

Reply:
I see, maybe in a follow up change we could create a class RandomFact where we could encapsulate this. Thoughts?